creator/nx_writer_xpcs.py

```diff
@@ -44,8 +44,8 @@ def go(self):
     # xpcs_md = {"g2": "g2_value",
     #            "g2_stderr": "g2_stderr_value",
     #            "tau": "tau_value",
-    #            "g2_partials_twotime": "g2_partials_twotime_values",
-    #            "g2_twotime": "g2_twotime_values",
+    #            "g2_from_two_time_corr_func_partials": "g2_partials_twotime_values",
+    #            "g2_from_two_time_corr_func": "g2_twotime_values",
     #            #TODO figure out how to access large amount of data
```

Comment:
Has this TODO item been resolved?

Reply:
I think at some point we discussed what array shape we would prefer but I can't recall how we wanted to deal with large numbers of g2 curves. I think we kind of left it open to whatever software would read the XPCS file. This TODO is probably for v2?